public/app/features/alerting/unified/group-details/GroupDetailsPage.tsx

```diff
@@ -78,6 +80,7 @@ function GroupDetailsPage() {
   );
   const { t } = useTranslate();
 
+  const ruleSourceName = dsFeatures?.rulerConfig?.dataSourceName;
```

Comment:
I think it won't work for vanilla Prometheus data sources. There won't be `rulerConfig` object for those
I think we need to use a different function to get a DS name from UID. Perhaps `getDataSourceByUid`?

Reply:
Good point! I'll fix that 🤔